src/controllers/updateJiraIssues.gs

```diff
@@ -67,16 +73,13 @@ function getDataForJiraUpdateFromSheet_() {
   return result;
 }
 
-/**
- * Finds the list of valid JIRA fields which can be edited
- * @returns {array} an array of built in and user selected custom fields
- */
-function getValidFieldsToEditJira_() {
-  var validFields = {};
-  var userSelectedcustomFields = getCustomFields(CUSTOMFIELD_FORMAT_SEARCH);
-  var systemFields = ISSUE_COLUMNS;
-  validFields = extend(validFields, userSelectedcustomFields);
-  validFields = extend(validFields, systemFields);
-  return validFields;
+/* Dialog: Update Fields in Jira Issues from Spreadsheet - END */
```

Comment:
moved getValidFieldsToEditJira_ to IssueFields - it feels more model related than view/controller